packages/rs-dpp/src/errors/consensus/basic/identity/invalid_identity_credit_withdrawal_transition_output_script_error.rs

```diff
@@ -29,6 +29,7 @@ impl InvalidIdentityCreditWithdrawalTransitionOutputScriptError {
     }
 
     pub fn output_script(&self) -> CoreScript {
+        // TODO: We shouldn't clone in getter 🤦
```

Comment:
It's fine for an error.

Reply:
It doesn't matter what it is. Hidden clone is a bad practice.